Automattic/jetpack
# Social | Initial state: Migrate `useAdminUiV1` feature flag to script data

Extracted out of #38952, this PR migrates on the `useAdminUiV1` feature flag and other feature flags will be migrated in the follow up PRs.

## Proposed changes:

* Migrate `useAdminUiV1` feature flag to script data

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* On Simple and Atomic sites, goto post editor
* Confirm that the editor loads fine
* Confirm that the connection management links point to Calypso
* For Jetpack sites, goto post editor
* Confirm that connection management buttons open the new connection management UI
* Goto Jetpack and Social admin page
* Confirm that the connection buttons open the modal instead of pointing to calypso
* Go to `/wp-admin/admin.php?page=jetpack#/recommendations/publicize`
* Confirm that the CTA "View Jetpack Social settings" links to the social settings section
* Goto `/wp-admin/admin.php?page=jetpack#/recommendations/unlimited-sharing-activated`
* Confirm that the CTA points to the social settings section
